js/client-battle.js

```diff
@@ -421,8 +421,20 @@
 			}
 			return '<button name="openTimer" class="button timerbutton' + timerTicking + '"><i class="fa fa-hourglass-start"></i> ' + time + '</button>';
 		},
+		timerInterval: 0,
+		timerTick: function () {
+			if (typeof this.battle.kickingInactive == "number") this.battle.kickingInactive--;
```

Comment:
(But, otherwise, I approve of this method of detecting what state the timer is in)